### DISCUSSION TITLE
Looking for a consultant for training

### DISCUSSION THREAD

**Discussion by dscisci** [jgraph/drawio#4214] [Q&A]
Hello there - I run a team of Solution Engineers and we're recently added draw.io as an integration to our product.  I'm looking to find an expert in various types of technical diagramming to give us training on using draw.io for these tasks.  We've tried in various freelance platforms to find someone and have not been successful.  This is of course, a paid opportunity. Please let me know if you know of any relevant resources. Thanks!

**Comment by ZsoltMONOLITE** [comment]
Hey,

The documentation is realy good, and the templates are good as well.

Regarding diagramming best practice i found this website, it has some "101" rules
https://www.fmc-modeling.org/

Hope this helps
